Containerfile.bytecode

```diff
@@ -1,13 +1,5 @@
 FROM scratch
 
-ARG PROGRAM_NAME
-ARG BPF_FUNCTION_NAME
-ARG PROGRAM_TYPE
-ARG BYTECODE_FILENAME
-ARG KERNEL_COMPILE_VER
+ARG BYTECODE_FILE
 
-COPY  $BYTECODE_FILENAME /
-LABEL io.ebpf.program_type $PROGRAM_TYPE
-LABEL io.ebpf.filename $BYTECODE_FILENAME
-LABEL io.ebpf.program_name $PROGRAM_NAME
-LABEL io.ebpf.bpf_function_name $BPF_FUNCTION_NAME
```

Comment:
If the bpf_function_name is removed, we'll need to modify the validation [here](https://github.com/bpfman/bpfman/blob/main/bpfman/src/types.rs#L630).

Reply:
For backwards compatibility, I was thinking of a check like the following:
```
if user provides a name, use it,
else if container has a name, use it
else, fail -- no name provided.
```

Reply:
I added a section/unit test for backwards compatibility